### PR TITLE
Tools: Test: Audio: Add possibility to test ASRC with src_test.m

### PR DESCRIPTION
**Pull request by singalsu**

### CODE REVIEW

File: tools/test/audio/src_test.m [hunk 1]
```diff
@@ -435,7 +441,7 @@
 %%
 
 function test = test_defaults_src(t)
-test.comp = 'src';
+test.comp = t.comp;
```
Comment (ShriramShastry) [inline tools/test/audio/src_test.m:444]: In function `test_run_src(test, t)`, t is unused.

**Review by ShriramShastry — APPROVED** [review]
I reviewed the Matlab changes and added one minor review comment, but the improvements seems to be good to me.
Thank you!

File: tools/test/audio/src_test.m (hunk 1 above)
Comment (lgirdwood) [reply tools/test/audio/src_test.m:444]: @singalsu pls fix in subsequent PR. I think this is a separate issue.

**Review by lgirdwood — APPROVED** [review]

File: tools/test/audio/src_test.m (hunk 1 above)
Comment (singalsu) [reply tools/test/audio/src_test.m:444]: Thanks, the fix is now in #8486.

**Pull request merged**